src/main/java/me/coley/cafedude/io/AttributeWriter.java

```diff
@@ -252,11 +255,115 @@ public AttributeWriter(ClassFile clazz) {
 				case Constants.Attributes.SOURCE_ID:
 					break;
 				case Constants.Attributes.STACK_MAP_TABLE:
+					StackMapTableAttribute stackMapTable =
+						(StackMapTableAttribute) attribute;
+					writeStackMapTable(out, stackMapTable);
 					break;
 				default:
 					break;
 			}
 		}
 		return baos.toByteArray();
 	}
+	
+	private void writeVerificationType(
+		DataOutputStream out,
+		StackMapTableAttribute.TypeInfo type
+	) throws IOException {
+		if (type instanceof StackMapTableAttribute.TopVariableInfo) {
+			out.writeByte(0);
+		} else if (type instanceof StackMapTableAttribute.IntegerVariableInfo) {
+			out.writeByte(1);
+		} else if (type instanceof StackMapTableAttribute.FloatVariableInfo) {
+			out.writeByte(2);
+		} else if (type instanceof StackMapTableAttribute.NullVariableInfo) {
+			out.writeByte(5);
+		} else if (
+			type instanceof StackMapTableAttribute.UninitializedThisVariableInfo
+		) {
```

Comment:
Why is the formatting like this for this case?

Reply:
Keeping below 80 columns

Reply:
80 is soooo tiny. We got decent monitors now, you can get away with more. 120 maybe if you wanna be conservative?